src/pdf/components/section-wrapper.astro

```diff
@@ -0,0 +1,15 @@
+---
+interface Props {
+  visible: boolean;
+}
+
+const { visible } = Astro.props;
+---
+
+{
+  visible && (
+    <div>
+      <slot />
+    </div>
+  )
+}
```

Comment:
Domyślam się, że chodziło tutaj o odtworzenie komponentu z wersji `web`. Niemniej, jako że ten wrapper jest dość pusty, to może wystarczy zamiast niego w `pdf.astro` dodać sprawdzanie czy sekcje są `visible` w ramach instrukcji warunkowej (`&&`)?